src/ViewLayer/ProgressBar/ProgressBar.h

```diff
@@ -1,9 +1,10 @@
 #pragma once
 
 #include <QWidget>
-#include <QPaintEvent>
 #include <QPainter>
```

Comment:
See if this can also be forward declared.

Reply:
Can use #include <QPen>